examples/plot_knn_s2.py

```diff
@@ -0,0 +1,68 @@
+"""Plot the result of a KNN classification on the sphere."""
+
+import logging
+import os
+
+import matplotlib.pyplot as plt
+
+import geomstats.backend as gs
+import geomstats.visualization as visualization
+from geomstats.geometry.hypersphere import Hypersphere
+from geomstats.learning.knn import KNearestNeighborsClassifier
+
+
+def main():
+    """Plot the result of a KNN classification on the sphere."""
+    sphere = Hypersphere(dimension=2)
+    sphere_distance = sphere.metric.dist
+
+    n_labels = 2
+    n_samples_per_dataset = 10
+    n_targets = 200
+
+    dataset_1 = sphere.random_von_mises_fisher(
+        kappa=10,
+        n_samples=n_samples_per_dataset)
+    dataset_2 = - sphere.random_von_mises_fisher(
+        kappa=10,
+        n_samples=n_samples_per_dataset)
+    training_dataset = gs.concatenate((dataset_1, dataset_2), axis=0)
+    labels_dataset_1 = gs.zeros([n_samples_per_dataset], dtype=int)
```

Comment:
Use gs.int instead of int: it will make the example more portable to pytorch and tensorflow later ;)

Reply:
I have the following error:
AttributeError: module 'geomstats.backend' has no attribute 'int'

Reply:
The nose2 test does not pass if I change int for gs.int.
Is it a recent change ?

Reply:
Do I add int in the backend ?